v2/concordium/types.proto

```diff
@@ -541,6 +541,9 @@ message AccountInfo {
   // the total amount that is actively staked or in cooldown (inactive stake).
   // This was introduced in node version 7.0.
   Amount available_balance = 12;
+  // A flag indicating whether the account is currently suspended or not. This
+  // was introduced in protocol version 8.
+  bool is_suspended = 13;
```

Comment:
This will be introduced in a specific node version (`8.0` probably) and not the protocol version, but the meaning of this will probably depend on the protocol version, which could be good to document as well. I assume that it will always be true for chains running protocol version 8 or earlier.

Reply:
That's a good that wasn't clear to me. I'll change the comment to reference node version 8.0.